Source/core/settings/ChromeSettingsProvider.js

```diff
@@ -67,11 +77,15 @@ ChromeSettingsProvider.prototype.get = function(key, def) {
  * returns carbon emission rate
  * default emission rate provided by:
  * https://www.epa.gov/sites/production/files/2016-02/documents/420f14040a.pdf
+ * find a credible source for public transport emission in kg/hr
```

Comment:
'hour', since time is provided in case of Public transport by google. I will also check what other map services provide.